src/components/utils/src/timer.cc

```diff
@@ -135,37 +59,75 @@ timer::Timer::Timer(const std::string& name, const TimerTask* task_for_tracking)
 timer::Timer::~Timer() {
   LOG4CXX_AUTO_TRACE(logger_);
   LOG4CXX_DEBUG(logger_, "Timer is to be destroyed " << name_);
-  Stop();
   sync_primitives::AutoLock auto_lock(task_lock_);
   DCHECK(task_);
   delete task_;
```

Comment:
@AGritsevich  Timer is Async and we cant guaranty that after you will delete this object from one thread, in other thread you wan't use it's methods, and if there will be null, it would be easier to debug such error.  So, plese set variable to NULL after delete in Timer